app/views/ad_templates/@responsive_footer/show.html.erb

```diff
@@ -1,12 +1,12 @@
 <% cache_if @campaign.persisted? && @creative.persisted?, [@campaign, @creative] do %>
   <div id="cf">
     <a data-href="campaign_url" class="cf-img-wrapper" target="_blank" rel="nofollow noopener">
-      <%= image_tag @creative.small_image.cloudfront_url, border: 0, height: 55, width: 55, class: "cf-img" rescue nil %>
+      <%= image_tag @creative.small_image.cloudfront_url, border: 0, height: 55, width: 55, class: "cf-img", alt: @creative.name rescue nil %>
     </a>
     <a data-href="campaign_url" class="cf-text" target="_blank" rel="nofollow noopener">
       <strong><%= @creative.headline %></strong>
       <span><%= @creative.body %></span>
-      <img data-src="impression_url">
+      <img data-src="impression_url", alt="">
```

Comment:
No comma required on this one. I might also update the `alt` text to, "Acknowledge ad impression".